### DISCUSSION TITLE
set precision

### DISCUSSION THREAD

**Discussion by TheFes** [frenck/spook#257] [Ideas] [answered]
Would it be possible a add a `set_precision` ~~service call~~ action to easily set the precision of multiple sensors in one go?

**Comment by TheFes** [comment]
Sorry for the wrong placement :)

**Comment by frenck** [comment, marked as answer]
Done, this shipped in Spook 5.1.0 as the `sensor.set_display_precision` action.

It takes a list of entities, so setting the precision on a pile of sensors in one go is exactly what it is for. It also resolves every entity before it changes any of them, so a typo halfway down your list fails the whole call instead of leaving you with half the sensors updated and nothing telling you which half.

One thing I should own: it has no documentation page yet. The action is there and works, I just never wrote the page. That is on my list.

../Frenck

<p>
  <a href="https://www.linkedin.com/in/frenck/"><img src="https://github.com/frenck/frenck/raw/main/images/linkedin.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://youtube.com/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/youtube.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://bsky.app/profile/frenck.social"><img src="https://github.com/frenck/frenck/raw/main/images/bluesky.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://fosstodon.org/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/mastodon.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.instagram.com/frenck/"><img src="https://github.com/frenck/frenck/raw/main/images/instagram.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.threads.net/@frenck"><img src="https://github.com/frenck/frenck/raw/main/images/threads.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.facebook.com/frenck.dev/"><img src="https://github.com/frenck/frenck/raw/main/images/facebook.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://x.com/frenck"><img src="https://github.com/frenck/frenck/raw/main/images/x.svg" width="18" height="18"></a>
  &nbsp;&nbsp;
  <a href="https://www.tiktok.com/@frenck.nl"><img src="https://github.com/frenck/frenck/raw/main/images/tiktok.svg" width="18" height="18"></a>
</p>

Blogging my personal ramblings at <a href="https://frenck.dev">frenck.dev</a>
